examples/release_highlights/plot_release_highlights_0_23_0.py

```diff
@@ -120,7 +119,6 @@
 # generally positively correlated with the first feature, with some noise.
 # Applying monotoinc constraints allows the prediction to capture the global
 # effect of the first feature, instead of fitting the noise.
-import numpy as np
```

Comment:
I think that the point here was to have codeblocks that could be copy-paster independently. @NicolasHug can confirm.

Hence, I think that this change should not be.

Reply:
yup, good catch. @willpeppo could you please put back the imports?

We usually want all imports at the top of the files, but the release highlights are a special case